command/init_test.go

```diff
@@ -52,19 +71,27 @@ func TestInitCommand_Run(t *testing.T) {
 				"github.com/sylviamoss/comment/packer-plugin-comment_v0.2.18_x5.0_linux_amd64_SHA256SUM":       "59031c50e0dfeedfde2b4e9445754804dce3f29e4efa737eead0ca9b4f5b85a5",
 			},
 			"h1:Q5qyAOdD43hL3CquQdVfaHpOYGf0UsZ/+wVA9Ry6cbA=",
-			`# cfg.pkr.hcl
-			packer {
-				required_plugins {
-					comment = {
-						source  = "github.com/sylviamoss/comment"
-						version = "v0.2.018"
-					}
-				}
-			}`,
-			cfg.dir("1"),
+			map[string]string{
```

Comment:
Would it be possible to test that trying to install a plugin that is not a multi-plugin fails?

Reply:
Sorry I realized I just asked a question with no context. 

Re: testing the failing case. I'm curious if it is something we could test for and if it makes sense to include in this PR. I know that we document that only muli-plugins work with `packer init` but I think that, if possible, a test case could help illustrate the case for folks reading the source code.

Reply:
Sure ! Good idea ! I'll open a new PR.